queries/javascript/highlights.scm

```diff
@@ -91,6 +91,14 @@
   (rest_parameter
     (identifier) @parameter))
 
+(variable_declarator                                                                                                                                                                                        
+    value: (arrow_function                                                                                                                                                                                    
+      parameter: (identifier) @parameter))
+
+(formal_parameters                                                                                                                                                                                          
+    (object_pattern                                                                                                                                                                                           
+      (shorthand_property_identifier) @parameter))
```

Comment:
@steelsojka i don't think this should be a property, object destructuring refers to object properties.

Reply:
I don't know. If it's destructured in the parameter list it's pretty much a parameter....  It's kind of like named parameters in a way. I'm kind of on the fence about it.